<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Transcriptional Regulation Underlying Long-term Sensitization in Aplysia - 5&nbsp; Encoding: What are the transcriptional changes induced by long-term sensitization training?</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./Maintenance.html" rel="next">
<link href="./Induction.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./Encoding.html"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Encoding: What are the transcriptional changes induced by long-term sensitization training?</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Transcriptional Regulation Underlying Long-term Sensitization in Aplysia</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Abstract</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Background.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Background: Sensitization Memory in <em>Aplysia</em></span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Induction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Induction: Why do only some experiences activate transcription to encode long-term sensitization memories?</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Encoding.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Encoding: What are the transcriptional changes induced by long-term sensitization training?</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Maintenance.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Maintenance: Does transcription contribute to the long-term maintenance of sensitization memory?</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Conclusions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Conclusions and Future Directions</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#the-early-wave-of-transcriptional-change-includes-up-regulation-of-transcription-factors-effector-genes-and-specific-isoforms-of-cam" id="toc-the-early-wave-of-transcriptional-change-includes-up-regulation-of-transcription-factors-effector-genes-and-specific-isoforms-of-cam" class="nav-link active" data-scroll-target="#the-early-wave-of-transcriptional-change-includes-up-regulation-of-transcription-factors-effector-genes-and-specific-isoforms-of-cam"><span class="header-section-number">5.1</span> The early wave of transcriptional change includes up-regulation of transcription factors, effector genes, and specific isoforms of CAM</a></li>
  <li><a href="#the-early-wave-of-transcriptional-response-produces-feedback-loops-that-help-stabilize-and-extend-induction-mechanisms" id="toc-the-early-wave-of-transcriptional-response-produces-feedback-loops-that-help-stabilize-and-extend-induction-mechanisms" class="nav-link" data-scroll-target="#the-early-wave-of-transcriptional-response-produces-feedback-loops-that-help-stabilize-and-extend-induction-mechanisms"><span class="header-section-number">5.2</span> The early wave of transcriptional response produces feedback loops that help stabilize and extend induction mechanisms</a></li>
  <li><a href="#the-late-wave-is-complex-and-involves-changes-potentially-related-to-neural-plasticity-protein-production-mrna-transport-and-meta-plasticity" id="toc-the-late-wave-is-complex-and-involves-changes-potentially-related-to-neural-plasticity-protein-production-mrna-transport-and-meta-plasticity" class="nav-link" data-scroll-target="#the-late-wave-is-complex-and-involves-changes-potentially-related-to-neural-plasticity-protein-production-mrna-transport-and-meta-plasticity"><span class="header-section-number">5.3</span> The late wave is complex and involves changes potentially related to neural plasticity, protein production, mRNA transport, and meta-plasticity</a></li>
  <li><a href="#encoding_both_waves" id="toc-encoding_both_waves" class="nav-link" data-scroll-target="#encoding_both_waves"><span class="header-section-number">5.4</span> A small and diverse set of transcripts participate in both the early and late waves of transcription</a></li>
  <li><a href="#the-late-wave-of-transcription-fades-before-forgetting-is-complete-but-some-transcripts-remain-regulated-long-after-sensitization-is-forgotten" id="toc-the-late-wave-of-transcription-fades-before-forgetting-is-complete-but-some-transcripts-remain-regulated-long-after-sensitization-is-forgotten" class="nav-link" data-scroll-target="#the-late-wave-of-transcription-fades-before-forgetting-is-complete-but-some-transcripts-remain-regulated-long-after-sensitization-is-forgotten"><span class="header-section-number">5.5</span> The late wave of transcription fades before forgetting is complete, but some transcripts remain regulated long-after sensitization is forgotten</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span id="sec-encoding-what-are-the-transcriptional-changes-induced-by-long-term-sensitization-training" class="quarto-section-identifier"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Encoding: What are the transcriptional changes induced by long-term sensitization training?</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>Successful long-term sensitization training protocols activate CREB1 and C/EBP while de-activating CREB2, triggering a transcriptional response that produces the cellular and synaptic changes required to encode a long-term sensitization memory.&nbsp; What, exactly, is this transcriptional response and how is it organized?</p>
<p>A framework for answering this question was developed from tracking not transcription but translation.&nbsp; Specifically, measurement of the rate of protein production in <em>Aplysia</em> nociceptors showed that serotonin produces two waves of increased translation: a small early wave evident within 1 hour that rapidly fades, and a larger late wave that arises more than 3 hours after the end of induction <span class="citation" data-cites="barzilai1989">(<a href="references.html#ref-barzilai1989" role="doc-biblioref">Barzilai et al. 1989</a>)</span>.&nbsp; The early wave is part of the immediate-early response, as it persists through treatments that block transcription.&nbsp; &nbsp;Subsequent studies have found that transcriptional changes are also organized into early and late waves, identifying transcripts that show rapid but transient changes in expression <span class="citation" data-cites="zwartjes1998">(<a href="references.html#ref-zwartjes1998" role="doc-biblioref">Zwartjes et al. 1998</a>)</span>, transcripts that show delayed changes in expression <span class="citation" data-cites="kuhl1992">(<a href="references.html#ref-kuhl1992" role="doc-biblioref">Kuhl et al. 1992a</a>)</span>, and some that bridge these phases of regulation <span class="citation" data-cites="hart2011 liu2011a">(<a href="references.html#ref-hart2011" role="doc-biblioref">Hart et al. 2011a</a>; <a href="references.html#ref-liu2011a" role="doc-biblioref">R.-Y. Liu et al. 2011a</a>)</span>.&nbsp;</p>
<p>The early and late waves have been further characterized via microarray studies conducted at different time points after long-term sensitization training (Figure 6).&nbsp; These studies estimate that the early wave is focused, involving only about 50 transcripts <span class="citation" data-cites="Herdegen2014Characterization">(<a href="references.html#ref-Herdegen2014Characterization" role="doc-biblioref">Herdegen, Holmes, et al. 2014</a>)</span>, that the late wave is complex, involving over 1,000 transcripts, and that very few (about 25) transcripts participate in both waves <span class="citation" data-cites="conte2017">(<a href="references.html#ref-conte2017" role="doc-biblioref">Conte et al. 2017a</a>)</span>.&nbsp; As sensitization memory is forgotten, the late wave also decays, but a handful of transcripts have been identified that remain regulated even beyond forgetting <span class="citation" data-cites="perez2018">(<a href="references.html#ref-perez2018" role="doc-biblioref">Perez et al. 2018a</a>)</span>.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Picture6.png" class="img-fluid figure-img" width="532"></p>
<figcaption class="figure-caption"><strong>Time Course of Behavioral and Transcriptional Changes Following Long-Term Sensitization.</strong> At top is the time-course of sensitization memory, showing both mean (solid lines) and individual (transparent lines) changes in defensive withdrawal durations on the trained (red) and untrained (sides) of Aplysia after long-term sensitization training (4 shocks). Sensitization memory is strongly expressed at 1 and 5 days after training, but fades back to baseline within 9 days. At bottom are log-fold changes in gene expression measured from microarray at 1 hour, 1 day, 5 days, and 7 days after the same long-term training protocol. Color-coding helps reveal the distinct patterns of regulation: There is an early-wave of transcripts up-regulated at 1 hour that then decays (blue), a large delayed wave of transcripts that show delayed up- or down-regulation at 1 day after training (purple with high transparency due to the large number of transcripts in this category), and a few transcripts that participate in both waves (green). Notably, most transcriptional changes resolve within 5 days of training, a time-point at which the expression of sensitization memory is still strong. A few transcripts, though, remain regulated even beyond forgetting (red). Behavioral data is replotted from Conte et al.&nbsp;(2017). Microarray data is synthesized from raw data posted from studies examining transcription 1 hour (Herdegen et al., 2014b), 1 day (Conte et al., 2017), and 7 days (Perez et al., 2018) after sensitization training.</figcaption>
</figure>
</div>
<p>This section summarizes what is known about the transcriptional changes forming the early and late waves of regulation after the induction of long-term sensitization and its <em>in vitro</em> analogs.&nbsp; Special focus is given to two interesting classes of transcriptional change: those that participate in both early and late waves of regulation, and those that persist even after sensitization is forgotten.</p>
<section id="the-early-wave-of-transcriptional-change-includes-up-regulation-of-transcription-factors-effector-genes-and-specific-isoforms-of-cam" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="the-early-wave-of-transcriptional-change-includes-up-regulation-of-transcription-factors-effector-genes-and-specific-isoforms-of-cam"><span class="header-section-number">5.1</span> The early wave of transcriptional change includes up-regulation of transcription factors, effector genes, and specific isoforms of CAM</h2>
<p>Long-term sensitization training produces an immediate-early response that includes up-regulation of c/EBP.&nbsp; Successful training stabilizes and extends this rapid phase of transcriptional change, but it remains transient, with c/EBP mRNA levels returning to baseline within 5 hours of induction. The early wave of transcription and translation detected after sensitization training consists of this immediate-early response and its subsequent stabilization and elaboration.</p>
<p>In addition to <em>c/EBP</em>, several specific transcripts in the early wave have been identified.&nbsp; Some are transcription factors, with sensitization training producing rapid increases in the expression of transcripts that encode CREB1 <span class="citation" data-cites="bartsch1998">(<a href="references.html#ref-bartsch1998" role="doc-biblioref">Bartsch et al. 1998</a>)</span> and a putative <em>Aplysia</em> homolog of C/EBPγ [aka Ig/EBP; <span class="citation" data-cites="herdegen2014a">Herdegen, Conte, et al. (<a href="references.html#ref-herdegen2014a" role="doc-biblioref">2014a</a>)</span>].&nbsp; In addition, several effector genes are regulated in the early wave of transcription.&nbsp; This includes transcripts encoding <em>Aplysia</em> homologs of ubiquitin hydrolase [uch; <span class="citation" data-cites="chain1999a">Chain et al. (<a href="references.html#ref-chain1999a" role="doc-biblioref">1999</a>)</span>; <span class="citation" data-cites="hegde1997">Hegde et al. (<a href="references.html#ref-hegde1997" role="doc-biblioref">1997a</a>)</span>], calmodulin <span class="citation" data-cites="zwartjes1998">(<a href="references.html#ref-zwartjes1998" role="doc-biblioref">Zwartjes et al. 1998</a>)</span>, the peptide neurotransmitter sensorin <span class="citation" data-cites="sun2001">(<a href="references.html#ref-sun2001" role="doc-biblioref">Sun, Wu, and Schacher 2001</a>)</span>, and an <em>Aplysia</em> Tolloid/BMP-1-like protein <span class="citation" data-cites="liu1997a">(<a href="references.html#ref-liu1997a" role="doc-biblioref">Q. R. Liu et al. 1997</a>)</span>.</p>
<p>There are probably additional transcriptional changes in the early wave, with a microarray analysis identifying over 50 additional transcripts showing rapid but transient regulation after sensitization training <span class="citation" data-cites="Herdegen2014Characterization">(<a href="references.html#ref-Herdegen2014Characterization" role="doc-biblioref">Herdegen, Holmes, et al. 2014</a>)</span>.&nbsp; Most of these additional transcripts have not yet been mapped to well-annotated gene models.&nbsp; In addition, a microarray analysis conducted immediately after <em>in vivo</em> serotonin exposure in <em>Aplysia kurodai</em> identified regulation of 4 novel transcripts that were validated via qPCR: up-regulation of transcripts predicted to encode homologs of matrilin, antistasin, and eukaryotic translation initiation factor 3 (EIF3) and down-regulation of a BAT1 <span class="citation" data-cites="lee2008">(<a href="references.html#ref-lee2008" role="doc-biblioref">Lee et al. 2008b</a>)</span>.&nbsp; Time-courses were not evaluated, but EIF3 is likely one of the rare transcripts regulated both rapidly and persistently; it is discussed in <span class="citation" data-cites="encoding_both_waves">(<a href="references.html#ref-encoding_both_waves" role="doc-biblioref"><strong>encoding_both_waves?</strong></a>)</span>.</p>
<p>One intriguing aspect of the early wave of transcription is that it can also produce changes in the expression of specific isoforms of the same gene.&nbsp; Specifically, work in cell culture shows that serotonin treatment shifts the expression of different isoforms of mRNA for an <em>Aplysia</em> homolog of NCAM immunoglobulin-related cell adhesion molecule (ApCAM), changing the balance of isoform expression in <em>Aplysia</em> nociceptors and motor neurons without changing overall mRNA levels across all ApCAM isoforms <span class="citation" data-cites="schacher2000">(<a href="references.html#ref-schacher2000" role="doc-biblioref">Schacher et al. 2000</a>)</span>.&nbsp; These changes in isoform expression require trans-synaptic interactions, as they only occur when sensory and motor neurons are co-cultured together.&nbsp; Changes in both overall and synaptic ApCAM expression play critical roles in producing <em>in vitro</em> long-term sensitization, contributing to both transcription factor activation <span class="citation" data-cites="lee2007">(<a href="references.html#ref-lee2007" role="doc-biblioref">S.-H. Lee et al. 2007</a>)</span> and synapse formation <span class="citation" data-cites="zhu1995">(<a href="references.html#ref-zhu1995" role="doc-biblioref">Zhu, Wu, and Schacher 1995</a>)</span>.&nbsp; The changes in ApCAM isoform expression produced by serotonin suggests that the trans-isoform measurement of gene expression that has so far been common in <em>Aplysia</em> research may miss important forms of transcriptional regulation, pointing to an important priority for future research.</p>
</section>
<section id="the-early-wave-of-transcriptional-response-produces-feedback-loops-that-help-stabilize-and-extend-induction-mechanisms" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="the-early-wave-of-transcriptional-response-produces-feedback-loops-that-help-stabilize-and-extend-induction-mechanisms"><span class="header-section-number">5.2</span> The early wave of transcriptional response produces feedback loops that help stabilize and extend induction mechanisms</h2>
<p>The immediate-early transcriptional response is initially fragile; it is extended and stabilized by complex interactions triggered by long-term training protocols.&nbsp; This can be observed in the dynamics of the key immediate-early expression of c/EBP.&nbsp; Even a short-term protocol (a single shock) is sufficient to produce a rapid increase in c/EBP transcription, but this fades in less than 1 hour, producing no lasting effects <span class="citation" data-cites="kopec2015a">(<a href="references.html#ref-kopec2015a" role="doc-biblioref">Kopec et al. 2015a</a>)</span>.&nbsp; With repeated shock, c/EBP becomes upregulated for several hours <span class="citation" data-cites="yamamoto1999">(<a href="references.html#ref-yamamoto1999" role="doc-biblioref">Yamamoto et al. 1999</a>)</span>.&nbsp; Treatments that prevent the stabilization of c/EBP expression impair the induction of long-term plasticity <span class="citation" data-cites="lee2012 mirisis2021 yim2006">(<a href="references.html#ref-lee2012" role="doc-biblioref">S.-H. Lee et al. 2012</a>; <a href="references.html#ref-mirisis2021" role="doc-biblioref">Mirisis, Kopec, and Carew 2021</a>; <a href="references.html#ref-yim2006" role="doc-biblioref">Yim et al. 2006</a>)</span>; treatments that extend its activation enable short-term protocols to produce lasting effects <span class="citation" data-cites="kim2006a lee2001a">(<a href="references.html#ref-kim2006a" role="doc-biblioref">Kim et al. 2006</a>; <a href="references.html#ref-lee2001a" role="doc-biblioref">J. a. Lee et al. 2001</a>)</span>.&nbsp;</p>
<p>Stabilization of the immediate-early response occurs, at least in part, through regulation of c/EBP activity and stability by MAPK isoforms.&nbsp; First, c/EBP is phosphorylated by ERK, and this enables the binding of C/EBP to ERE motifs in its target genes <span class="citation" data-cites="yamamoto1999">(<a href="references.html#ref-yamamoto1999" role="doc-biblioref">Yamamoto et al. 1999</a>)</span>.&nbsp; In addition, the p38 isoform of MAPK phosphorylates an <em>Aplysia</em> homolog of the RNA-binding protein embryonic lethal visual system (ELAV), protecting c/EBP mRNA from rapid degradation <span class="citation" data-cites="mirisis2021 yim2006">(<a href="references.html#ref-mirisis2021" role="doc-biblioref">Mirisis, Kopec, and Carew 2021</a>; <a href="references.html#ref-yim2006" role="doc-biblioref">Yim et al. 2006</a>)</span> that would otherwise occur from ARE/poly(U)-binding/degradation factor [AUF; <span class="citation" data-cites="lee2012a">Y.-S. Lee et al. (<a href="references.html#ref-lee2012a" role="doc-biblioref">2012</a>)</span>].&nbsp; These interactions integrate the extracellular signaling pathways contributing to ERK activation into the regulation of c/EBP: TrkB-receptor-activation is required for the initial increase in c/EBP transcription and NGF-B activation of P38 is required for the stabilization and extension of c/EBP transcription with long-term protocols <span class="citation" data-cites="kopec2015 mirisis2021">(<a href="references.html#ref-kopec2015" role="doc-biblioref">Kopec et al. 2015b</a>; <a href="references.html#ref-mirisis2021" role="doc-biblioref">Mirisis, Kopec, and Carew 2021</a>)</span>.&nbsp;</p>
<p>Stabilization of the early wave of transcription is also mediated by feedback between expressed genes and induction mechanisms.&nbsp; For example, the rapid increase in the expression of uch degrades the regulatory subunit of PKA, extending the time during which the catalytic domain is active <span class="citation" data-cites="hegde1997a">(<a href="references.html#ref-hegde1997a" role="doc-biblioref">Hegde et al. 1997b</a>)</span>.&nbsp; In addition, CREB1’s promotion of its own expression extends CREB1 activity <span class="citation" data-cites="mohamed2005">(<a href="references.html#ref-mohamed2005" role="doc-biblioref">Mohamed et al. 2005</a>)</span>; calmodulin mediates serotonergic activation of adenylyl cyclase <span class="citation" data-cites="lin2010">(<a href="references.html#ref-lin2010" role="doc-biblioref">Lin et al. 2010</a>)</span>; sensorin secretion activates autoreceptors that help activate ERK <span class="citation" data-cites="hu2004a">(<a href="references.html#ref-hu2004a" role="doc-biblioref">Jiang-Yuan Hu et al. 2004</a>)</span>, and tolloid regulates TGF-β signaling that is also important for ERK activation <span class="citation" data-cites="chin1999">(<a href="references.html#ref-chin1999" role="doc-biblioref">Chin et al. 1999</a>)</span>.&nbsp; Functional studies have confirmed that importance of these feedback loops, showing that the long-term persistence of serotonin-induced plasticity depends on upregulation of CREB1 <span class="citation" data-cites="liu2011c">(<a href="references.html#ref-liu2011c" role="doc-biblioref">R.-Y. Liu et al. 2011b</a>)</span> and uch <span class="citation" data-cites="hegde1997">(<a href="references.html#ref-hegde1997" role="doc-biblioref">Hegde et al. 1997a</a>)</span> as well as signaling from TGF-β, which is activated by tolloid <span class="citation" data-cites="chin1999 zhang1997">(<a href="references.html#ref-chin1999" role="doc-biblioref">Chin et al. 1999</a>; <a href="references.html#ref-zhang1997" role="doc-biblioref">Zhang et al. 1997</a>)</span>.&nbsp; For sensorin, serotonin produces not only a rapid but transient increase in its mRNA expression, but also export of sensorin mRNA to the neurites of nociceptors co-cultured with motor neurons, producing in increase in sensorin secretion that is essential for strengthening ERK activation and enabling serotonin to produce long-lasting plasticity&nbsp;<span class="citation" data-cites="hu2006">(<a href="references.html#ref-hu2006" role="doc-biblioref">J. Y. Hu, Wu, and Schacher 2006</a>)</span>. Some negative-feedback may also possible, as C/EBPγ lacks a transcriptional activation domain and in other animals has been shown to act as a negative inhibitor C/EBP function <span class="citation" data-cites="cooper1995">(<a href="references.html#ref-cooper1995" role="doc-biblioref">Cooper et al. 1995</a>)</span>; functional studies have not yet investigated this possibility.&nbsp;</p>
</section>
<section id="the-late-wave-is-complex-and-involves-changes-potentially-related-to-neural-plasticity-protein-production-mrna-transport-and-meta-plasticity" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="the-late-wave-is-complex-and-involves-changes-potentially-related-to-neural-plasticity-protein-production-mrna-transport-and-meta-plasticity"><span class="header-section-number">5.3</span> The late wave is complex and involves changes potentially related to neural plasticity, protein production, mRNA transport, and meta-plasticity</h2>
<p>Early proteomic screens showed that sensitization training and its <em>in vitro</em> analogs produce not only rapid changes in translation but also late-developing regulation, with several changes in expression becoming detectable only several hours after induction <span class="citation" data-cites="barzilai1989 castellucci1988">(<a href="references.html#ref-barzilai1989" role="doc-biblioref">Barzilai et al. 1989</a>; <a href="references.html#ref-castellucci1988" role="doc-biblioref">Castellucci et al. 1988</a>)</span>.&nbsp; Intense work subsequently identified two of these late-regulated genes: <em>Aplysia</em> homologs of the calreticulin <span class="citation" data-cites="kennedy1992">(<a href="references.html#ref-kennedy1992" role="doc-biblioref">Kennedy et al. 1992</a>)</span> and binding immunoglobin protein [BiP; <span class="citation" data-cites="kuhl1992b">Kuhl et al. (<a href="references.html#ref-kuhl1992b" role="doc-biblioref">1992b</a>)</span>], proteins involved in protein production and trafficking in the endoplasmic reticulum.&nbsp;</p>
<p>The late phase of transcription may be surprisingly complex, with microarray identifying 1,172 transcripts that show clear regulation that does not develop until 1 day after long-term sensitization training <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; In contrast to the early wave consisting almost exclusively of up-regulation, late-regulated transcripts were both up-regulated (722, 62%) and down-regulated (450, 38%).&nbsp; Given that the current draft of the <em>Aplysia</em> genome contains around 19,000 protein-coding gene models, the formation of a long-term sensitization memory is thus accompanied by regulation of an appreciable fraction of the genome.&nbsp; One caution to this conclusion is that proteomic screens have not uncovered the same degree of complexity, identifying just over 30 proteins showing clear changes in expression 1 or 2 days after treatment with serotonin <span class="citation" data-cites="monje2012 monje2013">(<a href="references.html#ref-monje2012" role="doc-biblioref">Francisco J. Monje et al. 2012</a>, <a href="references.html#ref-monje2013" role="doc-biblioref">2013a</a>)</span>.&nbsp; The reason for this discrepancy is not yet clear. It could be that not all transcriptional changes lead to translational changes, or it may be that changes in protein expression may be too subtle to easily detect, perhaps due to localization via transport and synapse-specific translation.&nbsp;</p>
<p>Although the late wave of transcriptional change is complex, some overall regulatory themes seem apparent.&nbsp; As might be expected, many late-regulated transcripts seem related to changes in neuronal signaling that could help express long-term sensitization memory.&nbsp; For example, there is a sharp up-regulation in mRNA and protein levels of an <em>Aplysia</em> homolog of synaptotagmin <span class="citation" data-cites="conte2017a monje2012">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>; <a href="references.html#ref-monje2012" role="doc-biblioref">Francisco J. Monje et al. 2012</a>)</span>, a protein which plays a conserved role in regulating neurotransmitter release <span class="citation" data-cites="fernández-chacón2001">(<a href="references.html#ref-fernández-chacón2001" role="doc-biblioref">Fernández-Chacón et al. 2001</a>)</span>.&nbsp; There is also an increase in transcription of an putative glutamate receptor <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; It is feasible that these changes contribute to increases in synaptic connectivity that help express sensitization.&nbsp; Indeed, two of the hallmarks of <em>in vitro</em> sensitization are long-term increases in nociceptor transmitter release and post-synaptic glutamate sensitivity <span class="citation" data-cites="trudeau1995 zhu1997">(<a href="references.html#ref-trudeau1995" role="doc-biblioref">Trudeau and Castellucci 1995</a>; <a href="references.html#ref-zhu1997" role="doc-biblioref">Zhu, Wu, and Schacher 1997</a>)</span>.&nbsp; Other potential changes relevant to neural signaling include transcripts annotated to encode proteins involved in synaptic targeting (e.g.&nbsp;semaphorin), growth-factor signaling (e.g.&nbsp;TGF-β receptor-associated protein), ion channels (e.g.&nbsp;volage-gated potassium channel subunit), and synaptic plasticity (e.g.&nbsp;menin-like, insulin-like growth factor 2, and fibroblast growth factor receptor 4 like).&nbsp; At the protein level, serotonin produces an increase in the expression of an <em>Aplysia</em> homolog of flotillin-1, which plays an important role in synapse formation and hippocampal-based memory formation in mammals <span class="citation" data-cites="monje2013a">(<a href="references.html#ref-monje2013a" role="doc-biblioref">Francisco J. Monje et al. 2013b</a>)</span>.</p>
<p>Beyond specific changes in neuronal signaling, many late transcriptional changes seem related to the infrastructure required to produce neuronal plasticity <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; That is, many of the transcriptional changes would be expected to foster the production, maturation, and trafficking of proteins, potentially providing the resources necessary to produce the signaling changes that encode sensitization.&nbsp; This includes up-regulation of putative <em>Aplysia</em> homologs of proteins related to transcription (e.g.&nbsp;RNA polymerase II elongation factor ELL-like), RNA processing (e.g.&nbsp;CUGBP Elav-like family member 3-B), translation (e.g.&nbsp;Translational activator GCN1) and post-translational processing (e.g.&nbsp;BiP).&nbsp; &nbsp;Especially notable is a widespread increase in the expression of translation initiation factors (eIF1, eIF2, eIF3, eIF4, and eIF5).&nbsp; Consistent with a global increase in protein production, studies in culture show a late-developing down-regulation of CREB2 mRNA produced through serotonergic activation of small non-coding RNAs that interact with the protein Piwi to repress CREB2 transcription <span class="citation" data-cites="rajasethupathy2012">(<a href="references.html#ref-rajasethupathy2012" role="doc-biblioref">Rajasethupathy et al. 2012</a>)</span>.</p>
<p>Although there seems to be a global increase in the infrastructure for protein production, this does not rule out the possibility of synapse-specific changes should mRNAs be transported and targeted for local translation.&nbsp; Consistent with this possibility, long-term sensitization also produces delayed up-regulation of several transcripts annotated to encode heterogeneous nuclear ribonucleoproteins and one annotated to encode a KIF protein (GenBank: EB225867.1); some of these changes are also detectable in proteomic screens <span class="citation" data-cites="monje2012">(<a href="references.html#ref-monje2012" role="doc-biblioref">Francisco J. Monje et al. 2012</a>)</span>.&nbsp; &nbsp;As these proteins are all implicated in neuronal mRNA transport <span class="citation" data-cites="hirokawa2006">(<a href="references.html#ref-hirokawa2006" role="doc-biblioref">Hirokawa 2006</a>)</span>, regulation of their expression could relate to targeting translational changes to specific synapses <span class="citation" data-cites="wang2010">(<a href="references.html#ref-wang2010" role="doc-biblioref">Wang et al. 2010</a>)</span>.</p>
<p>A fourth theme of the late phase of transcription is meta-plasticity: transcriptional changes that could feasibly be involved in resetting thresholds for additional learning and plasticity.&nbsp; Specifically, microarray analysis indicates delayed down-regulation of several transcripts associated with the induction of long-term sensitization <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; This includes decreases in the expression of 3 of the 4 known <em>Aplysia</em> homologs of adenylate cyclase, a type-4 cAMP-specific phosphodiesterase (PDE4) critical to the activation of cAMP (Park, 2005), and a putative serotonin receptor transcript.&nbsp; This set of transcriptional changes suggests the possibility that long-term sensitization makes it harder to induce additional sensitization.&nbsp; If sensitization produces a new learning context, this could help explain reconsolidation effects that have been observed after training <span class="citation" data-cites="cai2012">(<a href="references.html#ref-cai2012" role="doc-biblioref">Cai et al. 2012</a>)</span>.</p>
</section>
<section id="encoding_both_waves" class="level2" data-number="5.4">
<h2 data-number="5.4" class="anchored" data-anchor-id="encoding_both_waves"><span class="header-section-number">5.4</span> A small and diverse set of transcripts participate in both the early and late waves of transcription</h2>
<p>Although long-term sensitization produces distinct early and late waves of transcription, some transcripts participate in both phases of regulation, showing changes in expression that begin rapidly but also persist for at least 1 day after training. &nbsp;This is likely to be a rare form of regulation, with microarray analysis after sensitization training identifying only 25 potential “early-but-persistent” transcripts <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; Those that are well characterized are diverse, including transcription factors (CREB1 and Egr), synaptic proteins (synapsin), regulators of gene expression (eIF3 and TOB1), and modulatory peptide transmitters (FMRFa). &nbsp;&nbsp;</p>
<p>The early-but-persistent transcripts are of special interest, as their continued regulation suggests they may play some essential role in long-term sensitization.&nbsp; In particular, transcriptional changes involved in maintaining sensitization memory would be expected to show this type of stable regulation.&nbsp;&nbsp; Functional analysis of several “early-but-persistent” bears out this interest, demonstrating essential roles in <em>in vitro</em> analogs of sensitization.</p>
<p>One of the first “early-but-persistent” transcripts to be identified was synapsin <span class="citation" data-cites="hart2011a">(<a href="references.html#ref-hart2011a" role="doc-biblioref">Hart et al. 2011b</a>)</span>, a protein that regulates synaptic vesicles.&nbsp; In culture, repeated pulses of serotonin produce an immediate increase in synapsin mRNA that decays to a moderate down-regulation at 12 hours and then rises back to strong up-regulation within 1 day, indicating that synapsin is part of both the early and late waves of transcription.&nbsp; These changes are likely mediated by activation of CREB1, as serotonin promotes binding of CREB1 to a CRE-like site in the <em>synapsin</em> promoter.&nbsp; Transcriptional changes in synapsin are accompanied by a rapid rise in synapsin protein that within 24 hours is targeted specifically to synaptic varicosities in <em>Aplysia</em> nociceptors.&nbsp; Synapsin likely plays a critical role in long-term sensitization, as RNA interference timed to prevent changes in synapsin expression prevent serotonin from producing lasting synaptic facilitation.&nbsp;</p>
<p>eIF3 also shows early-but-persistent transcriptional activation.&nbsp; It was initially identified for showing a rapid increase in expression following <em>in vivo</em> serotonin exposure in intact animals <span class="citation" data-cites="lee2008a">(<a href="references.html#ref-lee2008a" role="doc-biblioref">Lee et al. 2008a</a>)</span>, and subsequent study showed it is also persistently up-regulated after long-term sensitization training <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; Consistent with the importance of this class of transcripts, work in culture shows that knockdown of eIF3 expression via RNA interference prevents the long-term synaptic effects of serotonin, while over-expression enables short-term protocols to produce lasting effects <span class="citation" data-cites="lee2008a">(<a href="references.html#ref-lee2008a" role="doc-biblioref">Lee et al. 2008a</a>)</span>.</p>
<p>A third well-characterized early-but-persistent transcript is an <em>Aplysia</em> homolog of early-growth response protein [Egr; <span class="citation" data-cites="cyriac2013">Cyriac et al. (<a href="references.html#ref-cyriac2013" role="doc-biblioref">2013</a>)</span>], a transcription factor that plays a key in long-term plasticity and memory in mammals <span class="citation" data-cites="poirier2008">(<a href="references.html#ref-poirier2008" role="doc-biblioref">Poirier et al. 2008</a>)</span>.&nbsp; Egr mRNA is constitutively expressed throughout the nervous system and is bi-directionally regulated by changes in neural activity.&nbsp;&nbsp; Long-term sensitization produces a rapid and sustained increase in Egr mRNA in <em>Aplysia</em> nociceptors <span class="citation" data-cites="herdegen2014b">(<a href="references.html#ref-herdegen2014b" role="doc-biblioref">Herdegen, Conte, et al. 2014b</a>)</span>.&nbsp; Although the function of Egr regulation has not yet been explored, the degree of up-regulation 1 day after training is correlated with the strength of memory expression at that time point <span class="citation" data-cites="cyriac2013">(<a href="references.html#ref-cyriac2013" role="doc-biblioref">Cyriac et al. 2013</a>)</span>.</p>
<p>Although already discussed as an early-wave transcript, CREB1 may be another transcript that participates in both early and late waves.&nbsp; CREB1 binds to its own promoter <span class="citation" data-cites="mohamed2005">(<a href="references.html#ref-mohamed2005" role="doc-biblioref">Mohamed et al. 2005</a>)</span>.&nbsp; In cell culture, repeated pulses of serotonin produce both a rapid increase in CREB1 mRNA that decays quickly and a delayed increase that lasts at least 1 day <span class="citation" data-cites="liu2008">(<a href="references.html#ref-liu2008" role="doc-biblioref">R.-Y. Liu et al. 2008</a>)</span>.&nbsp; This is accompanied by a long-lasting increase in the expression of CRE-containing promoters and is essential for serotonin to produce long-lasting increases in synaptic strength and excitability in <em>Aplysia</em> nociceptors <span class="citation" data-cites="liu2011d">(<a href="references.html#ref-liu2011d" role="doc-biblioref">R.-Y. Liu, Cleary, and Byrne 2011</a>)</span>.&nbsp; As discussed in the Maintenance section, the positive-feedback loop of CREB1 may serve not only to help stabilize the induction of long-term sensitization, but also to help maintain the persistence of sensitization memory.&nbsp; One caveat to this conclusion, however, is that in intact animals, CREB1 seems participate only in the first wave, with multiple samples failing to observe persistent changes in CREB1 mRNA after long-term sensitization training <span class="citation" data-cites="bonnick2012 conte2017a herdegen2014">(<a href="references.html#ref-bonnick2012" role="doc-biblioref">Bonnick et al. 2012</a>; <a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>; <a href="references.html#ref-herdegen2014" role="doc-biblioref">Herdegen, Conte, et al. 2014c</a>)</span>.&nbsp;</p>
<p>A surprising member of the early-but-persistent category is a transcript encoding the peptide neurotransmitter Phe-Met-Arg-Phe NH2 [FMRFa; <span class="citation" data-cites="schaefer1985">Schaefer et al. (<a href="references.html#ref-schaefer1985" role="doc-biblioref">1985</a>)</span>].&nbsp; This transcript is strongly up-regulated 1 hour and 1 day after long-term sensitization training in the pleural ganglia, which contain FMRF-amidergic neurons that provide inhibitory neuromodulation to defensive withdrawal circuits.&nbsp; This is accompanied by a late up-regulation of a FMRFa receptor in the pleural ganglia, suggesting an overall up-regulation of signaling from this transmitter system after sensitization.&nbsp; These changes are surprising because FRMFa can suppress memory <span class="citation" data-cites="fioravante2006">(<a href="references.html#ref-fioravante2006" role="doc-biblioref">Fioravante, Smolen, and Byrne 2006</a>)</span>, producing antagonistic effects on the expression of long-terms sensitization memory <span class="citation" data-cites="abrams1984">(e.g. <a href="references.html#ref-abrams1984" role="doc-biblioref">Abrams et al. 1984</a>)</span>.&nbsp; This suggests that in addition to activating transcriptional changes to encode sensitization memory, learning also activates signaling pathways that could eventually erode the expression of sensitization, a form of active forgetting <span class="citation" data-cites="berry2014">(<a href="references.html#ref-berry2014" role="doc-biblioref">Berry and Davis 2014</a>)</span>.&nbsp;</p>
<p>Microarray analysis has identified 21 additional transcripts which show clear regulation at both 1 hour and 1 day after long-term sensitization <span class="citation" data-cites="conte2017a">(<a href="references.html#ref-conte2017a" role="doc-biblioref">Conte et al. 2017b</a>)</span>.&nbsp; Some of these map to gene models that have not yet been well annotated.&nbsp; Those with annotations include ApGlyT2 (sodium- and chloride-dependent glycine-dependent transport 2), ApVPS36 (Vacuolar protein-sorting-associated protein 36-like), and a putative <em>Aplysia</em> homolog of TOB1 (transducer of ErbB-2).&nbsp; &nbsp;TOB1 is necessary for some forms of memory and is a key regulator of CPEB, a prion-like protein that plays a key role in the maintenance of the forms oflong-term plasticity that help encode sensitization memory (see <a href="Maintenance.html"><span>Chapter&nbsp;6</span></a>).&nbsp;</p>
</section>
<section id="the-late-wave-of-transcription-fades-before-forgetting-is-complete-but-some-transcripts-remain-regulated-long-after-sensitization-is-forgotten" class="level2" data-number="5.5">
<h2 data-number="5.5" class="anchored" data-anchor-id="the-late-wave-of-transcription-fades-before-forgetting-is-complete-but-some-transcripts-remain-regulated-long-after-sensitization-is-forgotten"><span class="header-section-number">5.5</span> The late wave of transcription fades before forgetting is complete, but some transcripts remain regulated long-after sensitization is forgotten</h2>
<p>Unless training is extensive, long-term sensitization memory is forgotten, becoming progressively less likely to be recalled.&nbsp; The late wave of transcription also fades, but seems to collapse before forgetting is completed.&nbsp; Microarray analysis shows that less 1% of the transcripts identified in the late wave remain regulated just 5 days after training, a time point at which sensitization memory has partly decayed but remains quite strong <span class="citation" data-cites="rosiles2019">(<a href="references.html#ref-rosiles2019" role="doc-biblioref">Rosiles et al. 2019</a>)</span>.&nbsp; This held true even in a subset of animals selected for especially strong maintenance of sensitization memory.&nbsp;</p>
<p>Although the vast majority of transcriptional changes induced by sensitization training decay, a handful of transcripts show very persistent regulation that extends well beyond the behavioral expression of sensitization memory <span class="citation" data-cites="patel2018 perez2018a">(<a href="references.html#ref-patel2018" role="doc-biblioref">Patel et al. 2018</a>; <a href="references.html#ref-perez2018a" role="doc-biblioref">Perez et al. 2018b</a>)</span>.&nbsp; While initially identified via exploratory microarray analysis, beyond-forgetting regulation has now been confirmed via qPCR in multiple independent samples.&nbsp; This includes upregulation of FMRFa, BiP, and 3 un-annotated transcripts and down-regulation of a putative homolog of spectrin and an un-annotated transcript.&nbsp; All of these are late-wave transcripts except FMRFa, which, as discussed above, is a an early-but-persistent transcript expected to work <em>against</em> the expression of sensitization memory.&nbsp;</p>
<p>It is not yet clear why some transcriptional changes might persist after sensitization memory is forgotten.&nbsp; They could be involved in producing forgetting, in maintaining a limited engram to enable rapid re-learning, or could even have no functional consequences at all.&nbsp;</p>


<div id="refs" class="references csl-bib-body hanging-indent" role="list" style="display: none">
<div id="ref-abrams1984" class="csl-entry" role="listitem">
Abrams, T W, V F Castellucci, J S Camardo, E R Kandel, and P E Lloyd. 1984. <span>“Two Endogenous Neuropeptides Modulate the Gill and Siphon Withdrawal Reflex in <span><em>Aplysia</em></span> by Presynaptic Facilitation Involving cAMP-Dependent Closure of a Serotonin-Sensitive Potassium Channel.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 81 (24): 7956–60. <a href="https://doi.org/10.1073/pnas.81.24.7956">https://doi.org/10.1073/pnas.81.24.7956</a>.
</div>
<div id="ref-bartsch1998" class="csl-entry" role="listitem">
Bartsch, D, A Casadio, K A Karl, P Serodio, and E R Kandel. 1998. <span>“CREB1 Encodes a Nuclear Activator, a Repressor, and a Cytoplasmic Modulator That Form a Regulatory Unit Critical for Long-Term Facilitation.”</span> <em>Cell</em> 95 (2): 211–23.
</div>
<div id="ref-barzilai1989" class="csl-entry" role="listitem">
Barzilai, a, T E Kennedy, J D Sweatt, and E R Kandel. 1989. <span>“5-HT Modulates Protein Synthesis and the Expression of Specific Proteins During Long-Term Facilitation in Aplysia Sensory Neurons.”</span> <em>Neuron</em> 2 (6): 1577–86.
</div>
<div id="ref-berry2014" class="csl-entry" role="listitem">
Berry, Jacob A., and Ronald L. Davis. 2014. <span>“Active forgetting of olfactory memories in Drosophila.”</span> <em>Progress in Brain Research</em> 208: 39–62. <a href="https://doi.org/10.1016/B978-0-444-63350-7.00002-4">https://doi.org/10.1016/B978-0-444-63350-7.00002-4</a>.
</div>
<div id="ref-bonnick2012" class="csl-entry" role="listitem">
Bonnick, Kristine, Karla Bayas, Dmitry Belchenko, Ashly Cyriac, Michael Dove, Jamie Lass, Benora McBride, Irina E. Calin-Jageman, and Robert J. Calin-Jageman. 2012. <span>“Transcriptional Changes Following Long-Term Sensitization Training and In Vivo Serotonin Exposure in Aplysia Californica.”</span> Edited by Zhong-Ping Feng. <em>PLoS ONE</em> 7 (10): e47378. <a href="https://doi.org/10.1371/journal.pone.0047378">https://doi.org/10.1371/journal.pone.0047378</a>.
</div>
<div id="ref-cai2012" class="csl-entry" role="listitem">
Cai, Diancai, Kaycey Pearce, Shanping Chen, and David L. Glanzman. 2012. <span>“Reconsolidation of long-term memory in <span><em>Aplysia</em></span>.”</span> <em>Current biology: CB</em> 22 (19): 1783–88. <a href="https://doi.org/10.1016/j.cub.2012.07.038">https://doi.org/10.1016/j.cub.2012.07.038</a>.
</div>
<div id="ref-castellucci1988" class="csl-entry" role="listitem">
Castellucci, V. F., T. E. Kennedy, E. R. Kandel, and P Goelet. 1988. <span>“A Quantitative Analysis of 2-d Gels Identifies Proteins in Which Labeling Is Increased Following Long-Term Sensitization in Aplysia.”</span> <em>Neuron</em> 1 (4): 321–28. <a href="https://doi.org/10.1016/0896-6273(88)90080-3">https://doi.org/10.1016/0896-6273(88)90080-3</a>.
</div>
<div id="ref-chain1999a" class="csl-entry" role="listitem">
Chain, Daniel G., Andrea Casadio, Samuel Schacher, Ashok N. Hegde, Mireille Valbrun, Naoki Yamamoto, Alfred L. Goldberg, Dusan Bartsch, Eric R. Kandel, and James H. Schwartz. 1999. <span>“Mechanisms for Generating the Autonomous cAMP-Dependent Protein Kinase Required for Long-Term Facilitation in Aplysia.”</span> <em>Neuron</em> 22 (1): 147–56. <a href="https://doi.org/10.1016/S0896-6273(00)80686-8">https://doi.org/10.1016/S0896-6273(00)80686-8</a>.
</div>
<div id="ref-chin1999" class="csl-entry" role="listitem">
Chin, Jeannie, Annie Angers, Leonard J. Cleary, Arnold Eskin, and John H. Byrne. 1999. <span>“Tgf-Β1 in <span><em>Aplysia</em></span>: Role in Long-Term Changes in the Excitability of Sensory Neurons and Distribution of Tβr-II-Like Immunoreactivity.”</span> <em>Learning &amp; Memory</em> 6 (3): 317–30. <a href="https://doi.org/10.1101/lm.6.3.317">https://doi.org/10.1101/lm.6.3.317</a>.
</div>
<div id="ref-conte2017" class="csl-entry" role="listitem">
Conte, Catherine, Samantha Herdegen, Saman Kamal, Jency Patel, Ushma Patel, Leticia Perez, Marissa Rivota, Robert J. Calin-Jageman, and Irina E. Calin-Jageman. 2017a. <span>“Transcriptional Correlates of Memory Maintenance Following Long-Term Sensitization of Aplysia Californica.”</span> <em>Learning &amp; Memory</em> 24 (10): 502–15. <a href="https://doi.org/10.1101/lm.045450.117">https://doi.org/10.1101/lm.045450.117</a>.
</div>
<div id="ref-conte2017a" class="csl-entry" role="listitem">
———. 2017b. <span>“Transcriptional Correlates of Memory Maintenance Following Long-Term Sensitization of Aplysia Californica.”</span> <em>Learning &amp; Memory</em> 24 (10): 502–15. <a href="https://doi.org/10.1101/lm.045450.117">https://doi.org/10.1101/lm.045450.117</a>.
</div>
<div id="ref-cooper1995" class="csl-entry" role="listitem">
Cooper, Cathleen, Andrew Henderson, Steven Artandi, Nicole Avitahl, and Kathryn Calame. 1995. <span>“Ig/EBP (c/EBPγ) Is a Transdominant Negative Inhibitor of c/EBP Family Transcriptional Activators.”</span> <em>Nucleic Acids Research</em> 23 (21): 4371–77. <a href="https://doi.org/10.1093/nar/23.21.4371">https://doi.org/10.1093/nar/23.21.4371</a>.
</div>
<div id="ref-cyriac2013" class="csl-entry" role="listitem">
Cyriac, Ashly, Geraldine Holmes, Jamie Lass, Dmitry Belchenko, Robert J. Calin-Jageman, and Irina E. Calin-Jageman. 2013. <span>“An <span><em>Aplysia</em></span> Egr Homolog Is Rapidly and Persistently Regulated by Long-Term Sensitization Training.”</span> <em>Neurobiology of Learning and Memory</em> 102 (May): 43–51. <a href="https://doi.org/10.1016/j.nlm.2013.03.008">https://doi.org/10.1016/j.nlm.2013.03.008</a>.
</div>
<div id="ref-fernández-chacón2001" class="csl-entry" role="listitem">
Fernández-Chacón, Rafael, Andreas Königstorfer, Stefan H. Gerber, Jesús García, Maria F. Matos, Charles F. Stevens, Nils Brose, Josep Rizo, Christian Rosenmund, and Thomas C. Südhof. 2001. <span>“Synaptotagmin I Functions as a Calcium Regulator of Release Probability.”</span> <em>Nature</em> 410 (6824): 41–49. <a href="https://doi.org/10.1038/35065004">https://doi.org/10.1038/35065004</a>.
</div>
<div id="ref-fioravante2006" class="csl-entry" role="listitem">
Fioravante, Diasinou, Paul D Smolen, and John H Byrne. 2006. <span>“The 5-HT- and FMRFa-Activated Signaling Pathways Interact at the Level of the Erk MAPK Cascade: Potential Inhibitory Constraints on Memory Formation.”</span> <em>Neuroscience Letters</em> 396 (3): 235–40. <a href="https://doi.org/10.1016/j.neulet.2005.11.036">https://doi.org/10.1016/j.neulet.2005.11.036</a>.
</div>
<div id="ref-hart2011" class="csl-entry" role="listitem">
Hart, Anne K., Diasinou Fioravante, Rong-Yu Liu, Gregg A. Phares, Leonard J. Cleary, and John H. Byrne. 2011a. <span>“Serotonin-Mediated Synapsin Expression Is Necessary for Long-Term Facilitation of the Aplysia Sensorimotor Synapse.”</span> <em>Journal of Neuroscience</em> 31 (50): 18401–11. <a href="https://doi.org/10.1523/JNEUROSCI.2816-11.2011">https://doi.org/10.1523/JNEUROSCI.2816-11.2011</a>.
</div>
<div id="ref-hart2011a" class="csl-entry" role="listitem">
———. 2011b. <span>“Serotonin-Mediated Synapsin Expression Is Necessary for Long-Term Facilitation of the Aplysia Sensorimotor Synapse.”</span> <em>Journal of Neuroscience</em> 31 (50): 18401–11. <a href="https://doi.org/10.1523/JNEUROSCI.2816-11.2011">https://doi.org/10.1523/JNEUROSCI.2816-11.2011</a>.
</div>
<div id="ref-hegde1997" class="csl-entry" role="listitem">
Hegde, Ashok N., Kaoru Inokuchi, Wanzheng Pei, Andrea Casadio, Mirella Ghirardi, Daniel G. Chain, Kelsey C. Martin, Eric R. Kandel, and James H. Schwartz. 1997a. <span>“Ubiquitin c-Terminal Hydrolase Is an Immediate-Early Gene Essential for Long-Term Facilitation in Aplysia.”</span> <em>Cell</em> 89 (1): 115–26. <a href="https://doi.org/10.1016/S0092-8674(00)80188-9">https://doi.org/10.1016/S0092-8674(00)80188-9</a>.
</div>
<div id="ref-hegde1997a" class="csl-entry" role="listitem">
———. 1997b. <span>“Ubiquitin c-Terminal Hydrolase Is an Immediate-Early Gene Essential for Long-Term Facilitation in <span><em>Aplysia</em></span>.”</span> <em>Cell</em> 89 (1): 115–26. <a href="https://doi.org/10.1016/S0092-8674(00)80188-9">https://doi.org/10.1016/S0092-8674(00)80188-9</a>.
</div>
<div id="ref-herdegen2014" class="csl-entry" role="listitem">
Herdegen, Samantha, Catherine Conte, Saman Kamal, Robert J. Calin-Jageman, and Irina E. Calin-Jageman. 2014c. <span>“Immediate and Persistent Transcriptional Correlates of Long-Term Sensitization Training at Different CNS Loci in Aplysia Californica.”</span> Edited by Björn Brembs. <em>PLoS ONE</em> 9 (12): e114481. <a href="https://doi.org/10.1371/journal.pone.0114481">https://doi.org/10.1371/journal.pone.0114481</a>.
</div>
<div id="ref-herdegen2014a" class="csl-entry" role="listitem">
———. 2014a. <span>“Immediate and Persistent Transcriptional Correlates of Long-Term Sensitization Training at Different CNS Loci in Aplysia Californica.”</span> <em>PLOS ONE</em> 9 (12): e114481. <a href="https://doi.org/10.1371/journal.pone.0114481">https://doi.org/10.1371/journal.pone.0114481</a>.
</div>
<div id="ref-herdegen2014b" class="csl-entry" role="listitem">
———. 2014b. <span>“Immediate and Persistent Transcriptional Correlates of Long-Term Sensitization Training at Different CNS Loci in Aplysia Californica.”</span> <em>PLOS ONE</em> 9 (12): e114481. <a href="https://doi.org/10.1371/journal.pone.0114481">https://doi.org/10.1371/journal.pone.0114481</a>.
</div>
<div id="ref-Herdegen2014Characterization" class="csl-entry" role="listitem">
Herdegen, Samantha, Geraldine Holmes, Ashly Cyriac, Irina E. Calin-Jageman, and Robert J Calin-Jageman. 2014. <span>“Characterization of the Rapid Transcriptional Response to Long-Term Sensitization Training in Aplysia Californica.”</span> <em>Neurobiology of Learning and Memory</em> 116 (December): 27–35. <a href="https://doi.org/10.1016/j.nlm.2014.07.009">https://doi.org/10.1016/j.nlm.2014.07.009</a>.
</div>
<div id="ref-hirokawa2006" class="csl-entry" role="listitem">
Hirokawa, Nobutaka. 2006. <span>“mRNA Transport in Dendrites: RNA Granules, Motors, and Tracks.”</span> <em>The Journal of Neuroscience</em> 26 (27): 7139–42. <a href="https://doi.org/10.1523/JNEUROSCI.1821-06.2006">https://doi.org/10.1523/JNEUROSCI.1821-06.2006</a>.
</div>
<div id="ref-hu2006" class="csl-entry" role="listitem">
Hu, J Y, F Wu, and S Schacher. 2006. <span>“Two Signaling Pathways Regulate the Expression and Secretion of a Neuropeptide Required for Long-Term Facilitation in <span><em>Aplysia</em></span>.”</span> <em>Journal of Neuroscience</em> 26: 1026–35. <a href="https://doi.org/10.1523/JNEUROSCI.4258-05.2006">https://doi.org/10.1523/JNEUROSCI.4258-05.2006</a>.
</div>
<div id="ref-hu2004a" class="csl-entry" role="listitem">
Hu, Jiang-Yuan, Leonard Glickman, Fang Wu, and Samuel Schacher. 2004. <span>“Serotonin Regulates the Secretion and Autocrine Action of a Neuropeptide to Activate MAPK Required for Long-Term Facilitation in <span><em>Aplysia</em></span>.”</span> <em>Neuron</em> 43 (3): 373–85. <a href="https://doi.org/10.1016/j.neuron.2004.07.011">https://doi.org/10.1016/j.neuron.2004.07.011</a>.
</div>
<div id="ref-kennedy1992" class="csl-entry" role="listitem">
Kennedy, T E, D Kuhl, a Barzilai, J D Sweatt, and E R Kandel. 1992. <span>“Long-Term Sensitization Training in <span><em>Aplysia</em></span> Leads to an Increase in Calreticulin, a Major Presynaptic Calcium-Binding Protein.”</span> <em>Neuron</em> 9 (6): 1013–24. <a href="https://doi.org/10.1016/0896-6273(92)90062-i">https://doi.org/10.1016/0896-6273(92)90062-i</a>.
</div>
<div id="ref-kim2006a" class="csl-entry" role="listitem">
Kim, Hyoung, Sue-Hyun Lee, Jin-Hee Han, Jin-A Lee, Ye-Hwang Cheang, Deok-Jin Chang, Yong-Seok Lee, and Bong-Kiun Kaang. 2006. <span>“A Nucleolar Protein ApLLP Induces ApC/EBP Expression Required for Long-Term Synaptic Facilitation in <span><em>Aplysia</em></span> Neurons.”</span> <em>Neuron</em> 49 (5): 707–18. <a href="https://doi.org/10.1016/j.neuron.2006.01.035">https://doi.org/10.1016/j.neuron.2006.01.035</a>.
</div>
<div id="ref-kopec2015" class="csl-entry" role="listitem">
Kopec, Ashley M, Gary T Philips, Thomas J Carew, Ashley M Kopec, Gary T Philips, and Thomas J Carew. 2015b. <span>“Distinct Growth Factor Families Are Recruited in Unique Spatiotemporal Domains During Long-Term Memory Formation in Aplysia Californica Distinct Growth Factor Families Are Recruited in Unique Spatiotemporal Domains During Long-Term Memory Formation in Apl.”</span> <em>Neuron</em> 86 (5): 1228–39. <a href="https://doi.org/10.1016/j.neuron.2015.04.025">https://doi.org/10.1016/j.neuron.2015.04.025</a>.
</div>
<div id="ref-kopec2015a" class="csl-entry" role="listitem">
———. 2015a. <span>“Distinct Growth Factor Families Are Recruited in Unique Spatiotemporal Domains During Long-Term Memory Formation in Aplysia Californica Distinct Growth Factor Families Are Recruited in Unique Spatiotemporal Domains During Long-Term Memory Formation in Apl.”</span> <em>Neuron</em> 86 (5): 1228–39. <a href="https://doi.org/10.1016/j.neuron.2015.04.025">https://doi.org/10.1016/j.neuron.2015.04.025</a>.
</div>
<div id="ref-kuhl1992" class="csl-entry" role="listitem">
Kuhl, D, T E Kennedy, A Barzilai, and E R Kandel. 1992a. <span>“Long-Term Sensitization Training in Aplysia Leads to an Increase in the Expression of BiP, the Major Protein Chaperon of the ER.”</span> <em>The Journal of Cell Biology</em> 119 (5): 1069–76. <a href="https://doi.org/10.1083/jcb.119.5.1069">https://doi.org/10.1083/jcb.119.5.1069</a>.
</div>
<div id="ref-kuhl1992b" class="csl-entry" role="listitem">
———. 1992b. <span>“Long-Term Sensitization Training in <span><em>Aplysia</em></span> Leads to an Increase in the Expression of BiP, the Major Protein Chaperon of the ER.”</span> <em>The Journal of Cell Biology</em> 119 (5): 1069–76. <a href="https://doi.org/10.1083/jcb.119.5.1069">https://doi.org/10.1083/jcb.119.5.1069</a>.
</div>
<div id="ref-lee2001a" class="csl-entry" role="listitem">
Lee, J a, H K Kim, K H Kim, J H Han, Y S Lee, C S Lim, D J Chang, T Kubo, and B K Kaang. 2001. <span>“Overexpression of and RNA Interference with the CCAAT Enhancer-Binding Protein on Long-Term Facilitation of <span><em>Aplysia</em></span> Sensory to Motor Synapses.”</span> <em>Learning &amp; Memory</em> 8 (4): 220–26. <a href="https://doi.org/10.1101/lm.40201">https://doi.org/10.1101/lm.40201</a>.
</div>
<div id="ref-lee2007" class="csl-entry" role="listitem">
Lee, Seung-Hee, Chae-Seok Lim, Hyungju Park, Jin-A. Lee, Jin-Hee Han, Hyoung Kim, Ye-Hwang Cheang, et al. 2007. <span>“Nuclear Translocation of CAM-Associated Protein Activates Transcription for Long-Term Facilitation in Aplysia.”</span> <em>Cell</em> 129 (4): 801–12. <a href="https://doi.org/10.1016/j.cell.2007.03.041">https://doi.org/10.1016/j.cell.2007.03.041</a>.
</div>
<div id="ref-lee2012" class="csl-entry" role="listitem">
Lee, S.-H., J. Shim, S.-L. Choi, N. Lee, C.-H. Lee, C. H. Bailey, E. R. Kandel, D.-J. Jang, and B.-K. Kaang. 2012. <span>“Learning-Related Synaptic Growth Mediated by Internalization of Aplysia Cell Adhesion Molecule Is Controlled by Membrane Phosphatidylinositol 4,5-Bisphosphate Synthetic Pathway.”</span> <em>Journal of Neuroscience</em> 32 (46): 16296–305. <a href="https://doi.org/10.1523/JNEUROSCI.1872-12.2012">https://doi.org/10.1523/JNEUROSCI.1872-12.2012</a>.
</div>
<div id="ref-lee2012a" class="csl-entry" role="listitem">
Lee, Yong-Seok, Sun-Lim Choi, Heejung Jun, Se-Jeong Yim, Jin-A. Lee, Hyoung F. Kim, Seung-Hee Lee, et al. 2012. <span>“AU-rich element-binding protein negatively regulates CCAAT enhancer-binding protein mRNA stability during long-term synaptic plasticity in <span><em>Aplysia</em></span>.”</span> <em>Proceedings of the National Academy of Sciences</em> 109 (38): 15520–25. <a href="https://doi.org/10.1073/pnas.1116224109">https://doi.org/10.1073/pnas.1116224109</a>.
</div>
<div id="ref-lee2008a" class="csl-entry" role="listitem">
Lee, Sun-Lim Choi, Tae-Hyung Kim, Jin-A Lee, Hyong Kyu Kim, Hyoung Kim, Deok-Jin Jang, et al. 2008a. <span>“Transcriptome Analysis and Identification of Regulators for Long-Term Plasticity in <span><em>Aplysia Kurodai</em></span>.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (47): 18602–7. <a href="https://doi.org/10.1073/pnas.0808893105">https://doi.org/10.1073/pnas.0808893105</a>.
</div>
<div id="ref-lee2008" class="csl-entry" role="listitem">
———, et al. 2008b. <span>“Transcriptome Analysis and Identification of Regulators for Long-Term Plasticity in Aplysia Kurodai.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 105 (47): 18602–7. <a href="https://doi.org/10.1073/pnas.0808893105">https://doi.org/10.1073/pnas.0808893105</a>.
</div>
<div id="ref-lin2010" class="csl-entry" role="listitem">
Lin, Chia Hsun Anthony, Masahiro Tomioka, Schreiber Pereira, Laurie Sellings, Yuichi Iino, and Derek van der Kooy. 2010. <span>“Insulin Signaling Plays a Dual Role in Caenorhabditis Elegans Memory Acquisition and Memory Retrieval.”</span> <em>J. Neurosci.</em> 30 (23): 8001–11. <a href="https://doi.org/10.1523/JNEUROSCI.4636-09.2010">https://doi.org/10.1523/JNEUROSCI.4636-09.2010</a>.
</div>
<div id="ref-liu1997a" class="csl-entry" role="listitem">
Liu, Q R, S Hattar, S Endo, K MacPhee, H Zhang, L J Cleary, J H Byrne, and A Eskin. 1997. <span>“A Developmental Gene (Tolloid/BMP-1) Is Regulated in Aplysia Neurons by Treatments That Induce Long-Term Sensitization.”</span> <em>The Journal of Neuroscience</em> 17 (2): 755–64. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8987797">http://www.ncbi.nlm.nih.gov/pubmed/8987797</a>.
</div>
<div id="ref-liu2008" class="csl-entry" role="listitem">
Liu, Rong-Yu, Diasinou Fioravante, Shreyansh Shah, and John H Byrne. 2008. <span>“cAMP Response Element-Binding Protein 1 Feedback Loop Is Necessary for Consolidation of Long-Term Synaptic Facilitation in <span><em>Aplysia</em></span>.”</span> <em>The Journal of Neuroscience</em> 28 (8): 1970–76. <a href="https://doi.org/10.1523/JNEUROSCI.3848-07.2008">https://doi.org/10.1523/JNEUROSCI.3848-07.2008</a>.
</div>
<div id="ref-liu2011a" class="csl-entry" role="listitem">
Liu, Rong-Yu, Shreyansh Shah, Leonard J Cleary, and John H Byrne. 2011a. <span>“Serotonin- and Training-Induced Dynamic Regulation of CREB2 in Aplysia.”</span> <em>Learning &amp; Memory</em> 18 (4): 245–49. <a href="https://doi.org/10.1101/lm.2112111">https://doi.org/10.1101/lm.2112111</a>.
</div>
<div id="ref-liu2011c" class="csl-entry" role="listitem">
———. 2011b. <span>“Serotonin- and Training-Induced Dynamic Regulation of CREB2 in Aplysia.”</span> <em>Learning &amp; Memory</em> 18 (4): 245–49. <a href="https://doi.org/10.1101/lm.2112111">https://doi.org/10.1101/lm.2112111</a>.
</div>
<div id="ref-liu2011d" class="csl-entry" role="listitem">
Liu, R.-Y., L. J. Cleary, and J. H. Byrne. 2011. <span>“The Requirement for Enhanced CREB1 Expression in Consolidation of Long-Term Synaptic Facilitation and Long-Term Excitability in Sensory Neurons of <span><em>Aplysia</em></span>.”</span> <em>Journal of Neuroscience</em> 31 (18): 6871–79. <a href="https://doi.org/10.1523/JNEUROSCI.5071-10.2011">https://doi.org/10.1523/JNEUROSCI.5071-10.2011</a>.
</div>
<div id="ref-mirisis2021" class="csl-entry" role="listitem">
Mirisis, Anastasios A., Ashley M. Kopec, and Thomas J. Carew. 2021. <span>“ELAV Proteins Bind and Stabilize C/EBP mRNA in the Induction of Long-Term Memory in <span><em>Aplysia</em></span>.”</span> <em>Journal of Neuroscience</em> 41 (5): 947–59. <a href="https://doi.org/10.1523/JNEUROSCI.2284-20.2020">https://doi.org/10.1523/JNEUROSCI.2284-20.2020</a>.
</div>
<div id="ref-mohamed2005" class="csl-entry" role="listitem">
Mohamed, Habib a, Weizhe Yao, Diasinou Fioravante, Paul D Smolen, and John H Byrne. 2005. <span>“cAMP-Response Elements in Aplysia Creb1, Creb2, and Ap-Uch Promoters: Implications for Feedback Loops Modulating Long Term Memory.”</span> <em>The Journal of Biological Chemistry</em> 280 (29): 27035–43. <a href="https://doi.org/10.1074/jbc.M502541200">https://doi.org/10.1074/jbc.M502541200</a>.
</div>
<div id="ref-monje2012" class="csl-entry" role="listitem">
Monje, Francisco J., Ruth Birner-Gruenberger, Barbara Darnhofer, Isabella Divisch, Daniela D. Pollak, and Gert Lubec. 2012. <span>“Proteomics Reveals Selective Regulation of Proteins in Response to Memory-Related Serotonin Stimulation in <span><em>Aplysia Californica</em></span> Ganglia.”</span> <em>Proteomics</em> 12 (3): 490–99. <a href="https://doi.org/10.1002/pmic.201100418">https://doi.org/10.1002/pmic.201100418</a>.
</div>
<div id="ref-monje2013" class="csl-entry" role="listitem">
Monje, Francisco J., Isabella Divisch, Marvie Demit, Gert Lubec, and Daniela D. Pollak. 2013a. <span>“Flotillin-1 Is an Evolutionary-Conserved Memory-Related Protein up-Regulated in Implicit and Explicit Learning Paradigms.”</span> <em>Annals of Medicine</em> 45 (4): 301–7. <a href="https://doi.org/10.3109/07853890.2013.770637">https://doi.org/10.3109/07853890.2013.770637</a>.
</div>
<div id="ref-monje2013a" class="csl-entry" role="listitem">
Monje, Francisco J, Isabella Divisch, Marvie Demit, Gert Lubec, and Daniela D Pollak. 2013b. <span>“Flotillin-1 Is an Evolutionary-Conserved Memory-Related Protein up-Regulated in Implicit and Explicit Learning Paradigms.”</span> <em>Annals of Medicine</em>, no. November 2012 (April): 1–7. <a href="https://doi.org/10.3109/07853890.2013.770637">https://doi.org/10.3109/07853890.2013.770637</a>.
</div>
<div id="ref-patel2018" class="csl-entry" role="listitem">
Patel, Ushma, Leticia Perez, Steven Farrell, Derek Steck, Athira Jacob, Tania Rosiles, Everett Krause, Melissa Nguyen, Robert J. Calin-Jageman, and Irina E. Calin-Jageman. 2018. <span>“Transcriptional Changes Before and After Forgetting of a Long-Term Sensitization Memory in <span><em>Aplysia Californica</em></span>.”</span> <em>Neurobiology of Learning and Memory</em> 155 (November): 474–85. <a href="https://doi.org/10.1016/j.nlm.2018.09.007">https://doi.org/10.1016/j.nlm.2018.09.007</a>.
</div>
<div id="ref-perez2018" class="csl-entry" role="listitem">
Perez, Leticia, Ushma Patel, Marissa Rivota, Irina E. Calin-Jageman, and Robert J Calin-Jageman. 2018a. <span>“Savings Memory Is Accompanied by Transcriptional Changes That Persist Beyond the Decay of Recall.”</span> <em>Learning &amp; Memory</em> 25 (1): 45–48. <a href="https://doi.org/10.1101/lm.046250.117">https://doi.org/10.1101/lm.046250.117</a>.
</div>
<div id="ref-perez2018a" class="csl-entry" role="listitem">
Perez, Leticia, Ushma Patel, Marissa Rivota, Irina E. Calin-Jageman, and Robert J. Calin-Jageman. 2018b. <span>“Savings Memory Is Accompanied by Transcriptional Changes That Persist Beyond the Decay of Recall.”</span> <em>Learning &amp; Memory</em> 25 (1): 45–48. <a href="https://doi.org/10.1101/lm.046250.117">https://doi.org/10.1101/lm.046250.117</a>.
</div>
<div id="ref-poirier2008" class="csl-entry" role="listitem">
Poirier, Roseline, Hélène Cheval, Caroline Mailhes, Sonia Garel, Patrick Charnay, Sabrina Davis, and Serge Laroche. 2008. <span>“Distinct Functions of Egr Gene Family Members in Cognitive Processes.”</span> <em>Frontiers in Neuroscience</em> 2 (1): 47–55. <a href="https://doi.org/10.3389/neuro.01.002.2008">https://doi.org/10.3389/neuro.01.002.2008</a>.
</div>
<div id="ref-rajasethupathy2012" class="csl-entry" role="listitem">
Rajasethupathy, Priyamvada, Igor Antonov, Robert Sheridan, Sebastian Frey, Chris Sander, Thomas Tuschl, and Eric&nbsp;R. Kandel. 2012. <span>“A Role for Neuronal piRNAs in the Epigenetic Control of Memory-Related Synaptic Plasticity.”</span> <em>Cell</em> 149 (3): 693–707. <a href="https://doi.org/10.1016/j.cell.2012.02.057">https://doi.org/10.1016/j.cell.2012.02.057</a>.
</div>
<div id="ref-rosiles2019" class="csl-entry" role="listitem">
Rosiles, Tania, Nguyen, Melissa, R Calin-Jageman, and Irina Calin-Jageman. 2019. <span>“Annual Meeting of the Society for Neuroscience.”</span> In. San Diego, CA. <a href="https://www.abstractsonline.com/pp8/#!/7883/presentation/68021">https://www.abstractsonline.com/pp8/#!/7883/presentation/68021</a>.
</div>
<div id="ref-schacher2000" class="csl-entry" role="listitem">
Schacher, Samuel, Fang Wu, Z Y Sun, and Denong Wang. 2000. <span>“Cell-Specific Changes in Expression of mRNAs Encoding Splice Variants of Aplysia Cell Adhesion Molecule Accompany Long-Term Synaptic Plasticity.”</span> <em>Journal of Neurobiology</em> 45 (3): 152–61. <a href="https://doi.org/10.1002/1097-4695(20001115)45:3<152::aid-neu3>3.0.co;2-0">https://doi.org/10.1002/1097-4695(20001115)45:3&lt;152::aid-neu3&gt;3.0.co;2-0</a>.
</div>
<div id="ref-schaefer1985" class="csl-entry" role="listitem">
Schaefer, Mark, Marina R. Picciotto, Thane Kreiner, Rashad Rudolf Kaldany, Ronald Taussig, and Richard H. Scheller. 1985. <span>“<span><em>Aplysia</em></span> Neurons Express a Gene Encoding Multiple FMRFamide Neuropeptides.”</span> <em>Cell</em> 41 (2): 457–67. <a href="https://doi.org/10.1016/S0092-8674(85)80019-2">https://doi.org/10.1016/S0092-8674(85)80019-2</a>.
</div>
<div id="ref-sun2001" class="csl-entry" role="listitem">
Sun, Zhong Yi, Fang Wu, and Samuel Schacher. 2001. <span>“Rapid Bidirectional Modulation of mRNA Expression and Export Accompany Long-Term Facilitation and Depression of Aplysia Synapses.”</span> <em>Journal of Neurobiology</em> 46 (1): 41–47. <a href="https://doi.org/10.1002/1097-4695(200101)46:1<41::AID-NEU4>3.0.CO;2-R">https://doi.org/10.1002/1097-4695(200101)46:1&lt;41::AID-NEU4&gt;3.0.CO;2-R</a>.
</div>
<div id="ref-trudeau1995" class="csl-entry" role="listitem">
Trudeau, L. E., and V. F. Castellucci. 1995. <span>“Postsynaptic Modifications in Long-Term Facilitation in Aplysia: Upregulation of Excitatory Amino Acid Receptors.”</span> <em>Journal of Neuroscience</em> 15 (2): 1275–84. <a href="https://doi.org/10.1523/JNEUROSCI.15-02-01275.1995">https://doi.org/10.1523/JNEUROSCI.15-02-01275.1995</a>.
</div>
<div id="ref-wang2010" class="csl-entry" role="listitem">
Wang, Dan Ohtan, Sang Mok Kim, Yali Zhao, Hongik Hwang, Satoru K Miura, Wayne S Sossin, and Kelsey C Martin. 2010. <span>“NIH Public Access”</span> 324 (5934): 1536–40. <a href="https://doi.org/10.1126/science.1173205.Synapse-">https://doi.org/10.1126/science.1173205.Synapse-</a>.
</div>
<div id="ref-yamamoto1999" class="csl-entry" role="listitem">
Yamamoto, N, a N Hegde, D G Chain, and J H Schwartz. 1999. <span>“Activation and Degradation of the Transcription Factor c/EBP During Long-Term Facilitation in Aplysia.”</span> <em>Journal of Neurochemistry</em> 73 (6): 2415–23. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10582601">http://www.ncbi.nlm.nih.gov/pubmed/10582601</a>.
</div>
<div id="ref-yim2006" class="csl-entry" role="listitem">
Yim, Se-Jeong, Yong-Seok Lee, Jin-A Lee, Deok-Jin Chang, Jin-Hee Han, Hyoung Kim, Hyungju Park, Heejung Jun, V. Narry Kim, and Bong-Kiun Kaang. 2006. <span>“Regulation of ApC/EBP mRNA by the <span><em>Aplysia</em></span> AU-Rich Element-Binding Protein, ApELAV, and Its Effects on 5-Hydroxytryptamine-Induced Long-Term Facilitation.”</span> <em>Journal of Neurochemistry</em> 98 (2): 420–29. <a href="https://doi.org/10.1111/j.1471-4159.2006.03887.x">https://doi.org/10.1111/j.1471-4159.2006.03887.x</a>.
</div>
<div id="ref-zhang1997" class="csl-entry" role="listitem">
Zhang, F, S Endo, L J Cleary, a Eskin, and J H Byrne. 1997. <span>“Role of Transforming Growth Factor-Beta in Long-Term Synaptic Facilitation in <span><em>Aplysia</em></span>.”</span> <em>Science</em> 275 (5304): 1318–20. <a href="https://doi.org/10.1126/science.275.5304.1318">https://doi.org/10.1126/science.275.5304.1318</a>.
</div>
<div id="ref-zhu1995" class="csl-entry" role="listitem">
Zhu, H., F. Wu, and S. Schacher. 1995. <span>“Changes in Expression and Distribution of Aplysia Cell Adhesion Molecules Can Influence Synapse Formation and Elimination in Vitro.”</span> <em>Journal of Neuroscience</em> 15 (6): 4173–83. <a href="https://doi.org/10.1523/JNEUROSCI.15-06-04173.1995">https://doi.org/10.1523/JNEUROSCI.15-06-04173.1995</a>.
</div>
<div id="ref-zhu1997" class="csl-entry" role="listitem">
———. 1997. <span>“Site-specific and sensory neuron-dependent increases in postsynaptic glutamate sensitivity accompany serotonin-induced long-term facilitation at <span><em>Aplysia</em></span> sensorimotor synapses.”</span> <em>The Journal of Neuroscience</em> 17 (13): 4976–86. <a href="https://doi.org/10.1523/JNEUROSCI.17-13-04976.1997">https://doi.org/10.1523/JNEUROSCI.17-13-04976.1997</a>.
</div>
<div id="ref-zwartjes1998" class="csl-entry" role="listitem">
Zwartjes, R E, H West, S Hattar, X Ren, F Noel, M Nunez-Regueiro, K MacPhee, et al. 1998. <span>“Identification of Specific mRNAs Affected by Treatments Producing Long-Term Facilitation in Aplysia.”</span> <em>Learning &amp; Memory</em> 4 (6): 478–95. <a href="https://doi.org/10.1101/lm.4.6.478">https://doi.org/10.1101/lm.4.6.478</a>.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./Induction.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Induction: Why do only some experiences activate transcription to encode long-term sensitization memories?</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./Maintenance.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Maintenance: Does transcription contribute to the long-term maintenance of sensitization memory?</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>